x/evm/client/cli/native_tx.go

```diff
@@ -56,6 +58,54 @@ func NativeSendTxCmd() *cobra.Command {
 	return cmd
 }
 
+func SendCastSeiAddrTxCmd() *cobra.Command {
```

Comment:
say `admin`'s true sei address is `seiABC` and true evm address is `0xXYZ`: `seid tx evm send <cast-addr> <amount> --from admin` would be an EVM tx with an EVM sig that decodes to the true evm address `0xXYZ`. The direct cast address of it `sei(0xXYZ)` doesn't have any fund to send because funds are under `seiABC`. This new command is a cosmos tx that sends from `seiABC` to `sei(0xXYZ)`

Reply:
understood, ty for exaplanation